x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="X.Org Xft library"
+DESCRIPTION="The X.Org project provides an open source implementation of the \
+X Window System. \
+The development work is being done in conjunction with the freedesktop.org \
+community. \
+The X.Org Foundation is the educational non-profit corporation whose \
+Board serves this effort, and whose Members lead this work."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="1989-95 GROUPE BULL
+	1998 Arnaud LE HORS
+	1996 Lorens Younes"
+# TODO: check all the licenses in the COPYING file in the next century.
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXft-$portVersion.tar.gz"
+CHECKSUM_SHA256="26cdddcc70b187833cbe9dc54df1864ba4c03a7175b2ca9276de9f05dce74507"
+SOURCE_DIR="libXft-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	libxft$secondaryArchSuffix = $portVersion
+	lib:libxft$secondaryArchSuffix = 4.11.0 compat >= 4
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libxrender$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxft${secondaryArchSuffix}_devel = $portVersion
+	devel:libxft$secondaryArchSuffix = 4.11.0 compat >= 4
+	"
+REQUIRES_devel="
+	libxft$secondaryArchSuffix == $portVersion base
+	devel:libx11$secondaryArchSuffix
+	devel:libxcb$secondaryArchSuffix
+	devel:libxau$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libx11$secondaryArchSuffix
+	devel:util_macros$secondaryArchSuffix
+	devel:kbproto$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libxrender$secondaryArchSuffix
+	devel:renderproto$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:automake
+	cmd:autoconf
+	cmd:autoreconf
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	autoreconf -i
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool library file
+	rm $libDir/libXft.la
+
+	prepareInstalledDevelLibs \
```

Comment:
Suggestion:
```diff
-	prepareInstalledDevelLibs \
-		libXft
+	prepareInstalledDevelLib libXft
-
 	fixPkgconfig
```
(Dropping the "s" in prepareInstalledDevelLib**s** is not required, but since we know that this recipe has only one lib and is likely to never have more than one lib, it would be slightly better to call `prepareInstalledDevelLib` instead of `prepareInstalledDevelLibs`.)

BTW, I have a few recipes which I have started some time ago but I need to do some polishing before I push them:
* x11-libs/libxscrnsaver
* x11-proto/printerproto
* x11-libs/libxp